src/tron/tronEngine.ts

```diff
@@ -896,6 +911,93 @@ export class TronEngine extends CurrencyEngine<TronTools> {
     return { contractJson, feeLimit, note }
   }
 
+  async makeFreezeTransaction(
+    action: TronFreezeAction
+  ): Promise<EdgeTransaction> {
+    const {
+      params: { nativeAmount, resource }
+    } = action
+
+    const contractJson = {
+      parameter: {
+        value: {
+          owner_address: base58ToHexAddress(this.walletLocalData.publicKey),
+          frozen_balance: parseInt(nativeAmount),
+          frozen_duration: 3,
```

Comment:
Where does this number come from?

Reply:
The the network enforced minimum duration in days. I'll add to networkInfo.